README.md

```diff
@@ -57,9 +57,12 @@ To load initial data from `data.sql` for testing you should add
 `spring.jpa.defer-datasource-initialization=true` and `spring.sql.init.mode=always`
 to application.properties
 
+### Authentication properties
+There are two authentication properties that must be set:
```

Comment:
I would say more like it can be overriden, because we already set some default values. I would write what default values are too.